master/getting-started/kubernetes/installation/hardway/istio-integration.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Istio integration
+canonical_url: 'https://docs.projectcalico.org/v3.8/getting-started/kubernetes/installation/hardway/istio-integration'
+---
+
+{{site.prodname}} policy integrates with [Istio](https://istio.io) to allow you to write policies that enforce against
+application layer attributes like HTTP methods or paths as well as against cryptographically secure identities. In this
+lab we will enable this integration and test it out.
+
+## Install FlexVolume driver
+
+{{site.prodname}} uses a FlexVolume driver to enable secure connectivity between Felix and the Dikastes container
+running in each pod.  It mounts a shared volume into which Felix inserts a Unix Domain Socket.
+
+On each node in the cluster, execute the following commands to install the FlexVolume driver binary.
+
+```
+sudo mkdir -p /usr/libexec/kubernetes/kubelet-plugins/volume/exec/nodeagent~uds
+sudo docker run --rm \
+  -v /usr/libexec/kubernetes/kubelet-plugins/volume/exec/nodeagent~uds:/host/driver \
+  calico/pod2daemon-flexvol:v3.8.0
```

Comment:
We shouldn't hard-code the v3.8.0 here. We can grab the version specific for this page through templating.

Reply:
So, I discussed this with @lxpollitt last week while writing and we agreed not to get this guide working with templates for the first iteration.  It's more than just the versions that really should be built using our templates.

Note also that things like the CNI Plugin are pulled from GitHub releases page, and there isn't a `master` binary available there.

I think we're going to have to live with Calico the hard way being frozen on a particular release (3.8.0 in this case) until we put in some more work to get it building against templates. And then, we'd probably want to put in some kind of testing to make sure template changes don't break Calico the hard way.

Reply:
ok